plone/recipe/codeanalysis/pep3101.py

```diff
@@ -5,6 +5,24 @@
 
 
 class PEP3101(CleanLines):
+    """
+    Checking for old style formatting is hard in the real world.
+    The logging module does not support PEP3101 yet, but then again,
+    you are not supposed to give the formatted string but the
+    format string and the parameters to be inserted.
+    I knewer found a reference for that but know that sentry
+    relies on it.
+    So now this check, looks for:
+    1: two string delimiter charactors following any number of whitespace and %
+    2. Beginning of line, any number of whitespace and %
+
+    This will ignore log messages that do not do inline message formatting,
+    and it will find multiline old style log formatting.
+
+    It will create false positives if you use % as a modulo operator
+    and calculation spans multiple lines with the modulo character beginning
+    a new line.
+    """
```

Comment:
No.
One of two conditions must be met:
at least two char delimiters before the %
only whitespace between line beginning and %

This would fail:
```
2 \
% 4
```